### DISCUSSION TITLE
Gallery left & right swipe often not recognized

### DISCUSSION THREAD

**Discussion by Phasetime** [Docile-Alligator/Infinity-For-Reddit#905] [Ideas]
When I'm in a gallery post, so multiple images that can be swipe through with a left and right swipe it often does not recognize my toucb inputs correctly and does not change the image. The normal "dragging of the image" along your touches doesnt get engaged.
I guess it could be because my swipe motions are not perfectly horizontal most of the time, but that really shouldn't be needed.

I also cross-posted to your subreddit with a video containing an example: https://www.reddit.com/r/Infinity_For_Reddit/comments/v93wrm/gallery_post_left_right_swipe_often_not_recognized/

Device information:
Google Pixel 3XL, Android 12 (build SP1A.210812.016.C1)
Using Gesture Navigation (swipe edges for back)
Infinity version 5.2.0

**Comment by Wladefant** [comment]
can confirm, that non horizontal swipes are not recognized as swipes and the image does not change

**Comment by JohnnyNoArms** [comment]
Can also confirm. I was about to make a feature request for this.

**Comment by woodpecs** [comment]
Just want to add I'm experiencing this phenomenon which is rather frustrating at times. However it appears to be more of an issue with imgur galleries where as reddit galleries behave much better even if the swipe isn't perfectly horizontal (I still occasionally miss-swipe at times). For the imgur galleries the swipes need to be perfectly horizontal or they will not work. This has made me realize how hard it is to actually swipe perfectly side to side. 

Not sure if this behavior is typical of all devices or only some? I have a OnePlus 7 Pro
